pkgs/applications/audio/vital/default.nix

```
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, makeWrapper
+, unzip
+
+, alsa-lib
+, libjack2
+, curl
+, xorg
+, libGL
+, freetype
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vital";
+  version = "1.5.5";
+
+  unpackCommand = ''
+    mkdir -p root
+    unzip $curSrc root
+  '';
+
+  src = fetchurl {
+    url = "https://builds.vital.audio/VitalAudio/vital/${builtins.replaceStrings ["."] ["_"] finalAttrs.version}/VitalInstaller.zip";
+    hash = "sha256-aPPH6EXz16W0SoOt62407yIVA98A55ZPfVofEyolLRM=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+    unzip
+  ];
+
+  buildInputs = [
+    alsa-lib
+    stdenv.cc.cc.lib
+    libGL
+    xorg.libSM
+    xorg.libICE
+    xorg.libX11
+    freetype
+    libjack2
+  ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    # copy each output to its destination (individually)
+    mkdir -p $out/{bin,lib/{clap,vst,vst3}}
+    for f in bin/Vital lib/{clap/Vital.clap,vst/Vital.so,vst3/Vital.vst3}; do
+      cp -r $f $out/$f
+    done
+
+    wrapProgram $out/bin/Vital \
+      --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath [ curl libjack2 ]}"
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = ./updater.sh;
+
+  meta = with lib; {
+    description = "Spectral warping wavetable synth.";
```

```suggestion
    description = "Spectral warping wavetable synth";
```
Please remove the trailing period.
https://nixos.org/manual/nixpkgs/stable/#var-meta-description

---

Ah, thanks! I wanted to remove it but that seemed informal. Glad to hear the docs support my instinct